docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+Camus
+================
+
+Camus is a pipeline from Kafka to HDFS.
+It is a MapReduce job that does distributed data loading from Kafka to HDFS.
+At LinkedIn, Camus is used to process tens of billions of messages per day.
+The confluent Camus distribution supports the latest CDH distribution.
+
+Camus includes the following features:
+    #. Automatic topic discovery
+    #. Avro schema management
+    #. Date partitioning
+
+All work is done within a single MapReduce job divided into three stages:
+    #. Setup stage:
+        Fetch available topics and partitions from Zookeeper and latest offsets from Kafka.
+    #. Hadoop stage:
+        Allocates topic pulls among a set number of MapReduce tasks. Each task does the following:
+            #. Fetch events from Kafka server and collect count statistics.
+            #. Write data to corresponding directories based on timestamps of events.
+            #. Produce count events and write to HDFS.
+            #. Store updated offsets in HDFS.
+    #. Cleanup stage:
+        Read counts from all tasks, aggregates the values,
+        and submits the results to Kafka for consumption by other systems.
+
+Quickstart 
+----------
+
+Assuming you have confluent platform installed.
+
+.. sourcecode:: bash
```

Comment:
Maybe also mention the location of the jar, e.g. "You'll need to specify the jar file's location when creating the Hadoop job. In Confluent's distribution, you can find the jar under `share/java/camus/` for the zip/tgz downloads or under `/usr/share/java/camus` when installed via yum or apt."